frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -161,8 +165,10 @@ export default function InteractiveWaveformModal({ selectedJob, setOpenInteracti
   const [openChangelog, setOpenChangelog] = useState(false);
   const [undoing, setUndoing] = useState(false);
   const [peakValleyWindows, setPeakValleyWindows] = useState({});
+  const [duplicatesPresent, setDuplicatesPresent] = useState(false);
```

Comment:
I don't think this needs to be here anymore, duplicates is handled in the InteractiveAnalysis component